### PR TITLE
Clarify `hours_to_expiration` for incremental models

### PR DESCRIPTION
**Pull request by dbeatty10**
## What are you changing in this pull request and why?

We want to clearly document that table options are only set on table creation.

@illiriks pointed out [here](https://github.com/dbt-labs/dbt-bigquery/issues/472) that table options (which include `hours_to_expiration`) are only set on table creation. This means that an incremental models that set its `hours_to_expiration` as `24` will expire 24 hours after its first run, even if it is incrementally updated every hour.

I read the style guide and added a **callout** within this PR. Open to it not being a callout if desired.

## Preview

🎩 
The preview is available here:
https://deploy-preview-2837--docs-getdbt-com.netlify.app/reference/resource-configs/bigquery-configs#controlling-table-expiration

Here's what the callout looks like:

image

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.

### CODE REVIEW

File: website/docs/reference/resource-configs/bigquery-configs.md [hunk 1]
```diff
@@ -690,6 +690,10 @@ from {{ ref('events') }}
 By default, dbt-created tables never expire. You can configure certain model(s)
 to expire after a set number of hours by setting `hours_to_expiration`.
 
+:::info Note
+The `hours_to_expiration` only applies to initial creation of the underlying table. For incremental models, the `hours_to_expiration` is **not** reset on incremental runs.
```
Comment (mirnawong1) [inline website/docs/reference/resource-configs/bigquery-configs.md]: great addition, Doug! added a non blocking suggestion @dbeatty10 to simplify the language. 

```suggestion
The `hours_to_expiration` only applies to initial creation of the underlying table. It doesn't reset for incremental models when they do another run.
```

**Review by mirnawong1 — DISMISSED** [review]
hey @dbeatty10 thank you so much for opening this up to clarify the language and reviewing the style guide!

I've left a non blocking suggestion to make the last sentence a bit clearer and shorter, however it is a non-blocker so please feel free to commit it or let me know your thoughts! ✨

File: website/docs/reference/resource-configs/bigquery-configs.md (hunk 1 above)
Comment (dbeatty10) [reply website/docs/reference/resource-configs/bigquery-configs.md]: Love it, thank you @mirnawong1 🤩

**Review by mirnawong1 — APPROVED** [review]
re-approving it for you @dbeatty10

**Comment by dbeatty10** [comment]
@mirnawong1 thanks for making it clearer and shorter -- exactly what I hoped would happen during the review!

Not only is the text better, the visual display is less busy also:

image

**Comment by mirnawong1** [comment]
it's an honor @dbeatty10 🙇

**Pull request merged**